crates/wdl-engine/src/eval/v1/task.rs

```diff
@@ -138,6 +141,34 @@ const OUTPUT_SCOPE_INDEX: ScopeIndex = ScopeIndex::new(1);
 /// visible.
 const TASK_SCOPE_INDEX: ScopeIndex = ScopeIndex::new(2);
 
+/// Returns the first entry in `map` that matches the provided keys.
+fn lookup_value<'a>(
+    map: &'a HashMap<String, Value>,
+    keys: &[&'static str],
+) -> Option<(&'a Value, &'static str)> {
+    keys.iter()
+        .find_map(|key| map.get(*key).map(|value| (value, *key)))
+}
+
+/// Parses an integer or byte-unit string into a byte count using the supplied
+/// `error_message` formatter when conversion fails.
+/// # Panics
```

Comment:
Sorry, the previous suggestion I had didn't have the proper formatting.

```suggestion

/// # Panics
```